components/app_bar/app_bar_binding.tsx

```diff
@@ -19,6 +19,8 @@ import {DoAppCallResult} from 'types/apps';
 
 import OverlayTrigger from 'components/overlay_trigger';
 
+const IMAGE_SIZE = '24px';
```

Comment:
this can just be a number

```suggestion
const IMAGE_SIZE = 24;
```

from MDN: "The intrinsic width of the image in pixels. Must be an integer without a unit"